src/test/java/org/openrewrite/java/migrate/lang/VarUsageTest.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.lang;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+import org.openrewrite.test.TypeValidation;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.version;
+
+class VarUsageTest implements RewriteTest {
+    public void defaults(RecipeSpec spec) {
+        spec.recipe(new VarKeyword())
+          .typeValidationOptions(new TypeValidation().methodInvocations(false));
```

Comment:
If we add the missing imports it should not be necessary to ignore type validation.
```suggestion
        spec.recipe(new VarKeyword());
```

Reply:
TBH I simply copied from your org.openrewrite.java.migrate.lang.StringFormattedTest :D 
Is there any reference or documentation when to use further customizations or do I missed downloading Sources with Javadoc?

Reply:
We have some details about testing recipes on these URLs; not specific to this particular case though:
- https://docs.openrewrite.org/authoring-recipes/recipe-testing#recipespec
If you feel you're missing some details feel free to let us know how we can improve! :)